superset-frontend/package.json

```diff
@@ -13,7 +13,7 @@
     "cover": "jest --coverage",
     "dev": "webpack --mode=development --colors --progress --debug --watch",
     "dev-server": "node --max_old_space_size=4096 ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --mode=development --progress",
-    "prod": "node --max_old_space_size=4096 webpack --mode=production --colors --progress",
+    "prod": "node --max_old_space_size=4096 ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --mode=production --colors --progress",
```

Comment:
do we want to be running the webpack-dev-server or just webpack for the prod build?

Reply:
oops good catch. Copy/paste oversight there.